pyworklistserver/worklist_server.py

```diff
@@ -1,40 +1,66 @@
 """ DICOM server implementation that responds to C-FIND requests"""
+import random
+import os
+import time
 from pynetdicom import AE
 from pynetdicom import evt
 from pynetdicom.sop_class import ModalityWorklistInformationFind
 
 from pyworklistserver import worklist
+from pyworklistserver import user_config
+from pyworklistserver import get_config
 
 class WorklistServer:
     """ Simple worklist server with support for one client. This
     implementation class is responsible for network communication with
     the client. """
 
-    def __init__(self, serverConfig, app_logger, seed, blocking):
+    def __init__(self, serverConfig, app_logger, blocking):
         self._config = serverConfig
+        self._config_values = get_config.GetConfig()
         self._logger = app_logger
         self._blocking = blocking
-        self._seed = seed
-        self._worklist_factory = worklist.RandomWorklist('ISO_IR 100', self._seed)
+        self._worklist_factory = worklist.RandomWorklist('ISO_IR 100', self._config_values)
         self._handlers = [
             (evt.EVT_C_FIND, self._on_find, [app_logger]),
         ]
         self._server = None
 
+    def get_seedNumber(self):
+        """Return Function for seed-value"""
+        if self._config_values.seed_Number == 0:
+            return random.randrange(1, 1000000)
+        return self._config_values.seed_Number
+
+    def setup_log_seed(self):
+        if os.path.exists("logfile.txt"):
+            os.remove("logfile.txt")
+        f = open("logfile.txt", "w+")

```

Comment:
Use 

    with(open(...)) as f:

This way, you don't need to think about closing it.